ztp/gitops-subscriptions/argocd/example/policygentemplates/common-ranGen.yaml

```diff
@@ -9,6 +9,15 @@ spec:
     common: "true"
   sourceFiles:
     # Create inform policy to validate configuration CRs that will be applied to all clusters
+    - fileName: ClusterVersion.yaml
+      policyName: "cluster-version-policy"
+      remediationAction: inform
+      spec:
+        channel: "4.10"
+        desiredUpdate:
+          force: false
+          version: "4.10"
+          upstream: "upstream"
```

Comment:
I don't think we should include the ClusterVersion CR in the original ZTP set of CRs. The cluster installation version will be controlled by the choice of ClusterImageSet in the SiteConfig. If the user wants to do an upgrade they should create a Policy with the ClusterVersion CR (can be via gitops ang PGT, just not part of the ZTP set of PGTs) and then manually create an upgrade CR to roll that out.

Reply:
ok so want to make sure I understand it correctly:
1. should it be its own set of crs that we only need for upgrades like for instance `upgradesCRs` is that what we are looking for? 
2. should it be part of the `sourceCRs` and have its own PGT that we could integrate in the platform upgrade flow?

Reply:
I don't have a problem with including the CR in source-crs. My request is that it not be a part of the reference PolicyGenTemplates that are used for ZTP (any of the PGTs under ztp/gitops-subscriptions/argocd/examples/policygentemplates). 
If at "day-N" a user wants to use a PGT to create this ClusterVersion CR they can do so because we've included it in source-crs, we're just not creating it as part of ZTP (because it isn't needed then),